src/backend/distributed/planner/function_call_delegation.c

```diff
@@ -257,6 +292,24 @@ TryToDelegateFunctionCall(Query *query)
 		return NULL;
 	}
 
+	(void) expression_tree_walker((Node *) funcExpr->args, contain_param_walker,
+								  &walkerParamContext);
+	if (walkerParamContext.hasParam)
+	{
+		if (walkerParamContext.paramKind == PARAM_EXTERN)
+		{
+			*hasExternParam = true;
+			ereport(DEBUG1, (errmsg("external parameter found in arguments to "
```

Comment:
Can you also add a test to hit this case?

Reply:
I think this is unreachable, since if this is parametrized then distribution column is also parametrized and we have returned earlier. Maybe we can change this to an Assert() in future after careful consideration, but this is ok for now